deepmd/pt/utils/learning_rate.py

```diff
@@ -3,14 +3,36 @@
 
 
 class LearningRateExp:
-    def __init__(self, start_lr, stop_lr, decay_steps, stop_steps, **kwargs):
-        """Construct an exponential-decayed learning rate.
+    def __init__(
+        self,
+        start_lr,
+        stop_lr,
```

Comment:
`stop_lr` may be made optional, user may not want to set it if decay_rat is set.

Reply:
Theoretically, one only needs to set there among `start_lr`, `stop_lr`, `decay_steps`, and `decay_rate`.

Reply:
I see, we can use `stop_lr` param instead of the `min_lr` param when users set `decay_rate` explicitly.
1. when users set `start_lr`, `decay_steps`, `stop_lr`, it will follow the traditional way to compute the `decay_rate`
2. when users set `start_lr`, `decay_steps`, `stop_lr` and `decay_rate` in pytorch, it will take the user-defined `decay_rate` and with the minimum learning rate to be `stop_lr`.

Reply:
I am not sure whether we should merge these two modes into one doc, or document them separately using `Variant`.